### ISSUE TITLE
Boringtun won't start if IPv6 is disabled

### ISSUE THREAD

**Issue by jait** [cloudflare/boringtun#71]
When running boringtun on a system where IPv6 is disabled, I get:
`Socket("Address family not supported by protocol")`
and the process exits.

Strace indicates that the reason is indeed IPv6 socket creation:
`socket(AF_INET6, SOCK_DGRAM, IPPROTO_IP) = -1 EAFNOSUPPORT (Address family not supported by protocol)`

**Comment by vkrasnov** [comment]
Yes, currently the expectation is that both v4 and v6 are supported. It will be fixed, thank you.

**Comment by creativetags** [comment]
Is IPv6 still required?

**Comment by perwigren** [comment]
> Is IPv6 still required?

Yes, apparently. I get the same error from strace in Docker on latest git master.
